NAMESPACE

```diff
@@ -63,7 +63,10 @@ export(
        "BinaryOption",
        "BarrierOption",
        ##--schedule.R
-       "Schedule"
+       "Schedule",
+       ##--FXDerivatives.R
+       "oneTouch",
+       "DNTVannaVolga"
```

Comment:
For the one Touch, it could be called AmericanBinaryOption, which is in line naming schemes. For the DNT, it could be called DoubleNoTouchOption, and specify in the documentation that it is using vanna-volga method for pricing ?